rosetta/failure/invalid_blockchain.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package failure
+
+import (
+	"fmt"
+)
+
+type InvalidBlockchain struct {
+	Description          Description
+	HaveBlockchain       string
+	AvailableBlockchains string
```

Comment:
If you want to call the field that, it should be a `[]string`.

Reply:
I followed suite with the names used before for the detail (`available_blockchains`). But, since we only have a single valid blockchain/network at the moment, I figured it was better to be consistent with other values/errors, so I just used singular with `have`/`want`.